common/Timer.cc

```diff
@@ -44,6 +44,8 @@ microseconds Timer::clock_gettime_coarse() {
 Timer::Timer(spdlog::logger &log, ConstExprStr name, FlowId prev, initializer_list<pair<ConstExprStr, string>> args,
              microseconds start, initializer_list<int> histogramBuckets)
     : log(log), name(name), prev(prev), self{0}, start(start) {
+    keepShortTimers = false;
```

Comment:
Let's not modify static field in instance methods

Reply:
It should probably have a separate initializer in this file.

Reply:
==, this should be a `private` `static` property on the `Timer` class.